samples/cellular/smp_client/src/main.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <ctype.h>
+#include <zephyr/drivers/gpio.h>
+#include <stdio.h>
+#include <modem/nrf_modem_lib.h>
+#include <app_event_manager.h>
+#include <date_time.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app_smp_client, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include <modem/lte_lc.h>
+#include <modem/modem_info.h>
+#include <nrf_modem_at.h>
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <dfu/dfu_target_smp.h>
+#include <mcumgr_smp_client.h>
+
+#if !defined(CONFIG_LTE_LINK_CONTROL)
+#error "Missing CONFIG_LTE_LINK_CONTROL"
+#endif
+
+#define APP_BANNER "Run SMP client"
+
+/* Client State Machine states */
+static enum client_state {
+	INIT,	       /* Init state  */
+	LTE_CONNECTED, /* LTE connection Online */
+	LTE_OFFLINE    /* LTE offline */
+} client_state = INIT;
+
+static void state_trigger_and_unlock(enum client_state new_state);
+
+static K_SEM_DEFINE(state_mutex, 0, 1);
+static K_MUTEX_DEFINE(lte_mutex);
+static bool modem_connected_to_network;
+
+#define RESET_NODE DT_NODELABEL(nrf52840_reset)
+
+#if DT_NODE_HAS_STATUS(RESET_NODE, okay)
+#define RESET_GPIO_CTRL  DT_GPIO_CTLR(RESET_NODE, gpios)
+#define RESET_GPIO_PIN   DT_GPIO_PIN(RESET_NODE, gpios)
+#define RESET_GPIO_FLAGS DT_GPIO_FLAGS(RESET_NODE, gpios)
+
+static int nrf52840_reset_api(void)
+{
+	int err;
+	const struct device *const port = DEVICE_DT_GET(RESET_GPIO_CTRL);
+
+	if (!device_is_ready(port)) {
+		LOG_ERR("Reset device not ready");
+		return -EIO;
+	}
+
+	/* Configure pin as output and initialize it to inactive state. */
+	err = gpio_pin_configure(port, RESET_GPIO_PIN,
+				 RESET_GPIO_FLAGS | GPIO_OUTPUT_INACTIVE);
+	if (err) {
+		LOG_ERR("Pin configure err:%d", err);
+		return err;
+	}
+
+	/* Reset the nRF52840 and let it wait until the pin is inactive again
+	 * before running to main to ensure that it won't send any data until
+	 * the H4 device is setup and ready to receive.
+	 */
+	err = gpio_pin_set(port, RESET_GPIO_PIN, 1);
```

Comment:
Use the correct define for this e.g. `GPIO_OUTPUT_ACTIVE`

Reply:
This is not working need to use 1